examples/ansible/kubeone/galaxy.yml

```diff
@@ -0,0 +1,12 @@
+namespace: kubermatic
+name: kubeone
+version: "0.10.0-alpha.0"
+readme: README.md
+authors:
+  - "Alexander Sowitzki <alexander@loodse.com>"
+license:
+    - "Apache-2.0"
+repository: https://github.com/kubermatic/kubeone
+documentation: https://github.com/kubermatic/kubeone
```

Comment:
You can eventually link to the `docs` folder here

Reply:
ok